### PR TITLE
Add chakra-ui to the Project

### PR DESCRIPTION
**Pull request by zubairahad**
Screenshot 2023-08-17 at 10 02 42 PM

### CODE REVIEW

**Comment by UmmarHamid** [comment]
@zubairahad The PR name should make some sense, so instead of "Chakraui", you can use something like 'Add chakra-ui to the Project'

**Comment by UmmarHamid** [comment]
@zubairahad The commit messages should be descriptive to let reviewers know what it does!
'see these changes. especially logo', it should be 'add logo for iust-portal'
We use future tense for commit messages, so instead of 'added logo', we use 'add logo'

**Comment by UmmarHamid** [comment]
@zubairahad Can you add some screenshots of the changes?

File: components/Logo/iustlogo.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+const iustlogo = () => {
+  return (
+    <div className='logo'>
+      <a href='google.com'>
+        <Image src='/iustlogo.png' alt='missing' width={200} height={65} />
+      </a>
+    </div>
+  );
+};
+export default iustlogo;
```
Comment (UmmarHamid) [inline components/Logo/iustlogo.tsx]: This should not be called iustlogo, Component names are always generic, you can call this Logo.
All components should use PascalCase

File: components/Logo/iustlogo.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Image from 'next/image';
+import Link from 'next/link';
+const iustlogo = () => {
+  return (
+    <div className='logo'>
+      <a href='google.com'>
```
Comment (UmmarHamid) [inline components/Logo/iustlogo.tsx]: href should be a prop instead of hardcoded string

File: components/Logo/iustlogo.tsx (hunk 1 above)
Comment (UmmarHamid) [inline components/Logo/iustlogo.tsx]:
```suggestion
import Image from 'next/image';
import Link from 'next/link';
const Logo = ({url, imgSrc, alt, width, height}) => {
  return (
    <div className='logo'>
      <a href={url}>
        <Image src={imgScr} alt={alt} width={width} height={height} />
      </a>
    </div>
  );
};
export default Logo;
```

**Comment by zubairahad** [comment]
Sir voh kaisaa??

File: styles/globals.css [hunk 3]
```diff
@@ -1,107 +1,4 @@
...
-  text-decoration: none;
-}
-
-@media (prefers-color-scheme: dark) {
-  html {
-    color-scheme: dark;
-  }
+.logo {
+  display: flex;
+  padding: 20px;
```
Comment (UmmarHamid) [inline styles/globals.css]: All the styling should be done using styled Components

File: pages/index.tsx [hunk 4]
```diff
@@ -1,114 +1,22 @@
...
-            <p>
-              Instantly deploy your Next.js site to a shareable URL
-              with&nbsp;Vercel.
-            </p>
-          </a>
-        </div>
+      <main>
+        <Iustlogo />
+        <Login />
       </main>
```
Comment (UmmarHamid) [inline pages/index.tsx]:
```suggestion
      <main>
        <Logo />
        <Button />
      </main>
```

File: components/Button/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+      </Button>
+      <Button rightIcon={<MdCall />} colorScheme='blue' variant='outline'>
+        Call us
+      </Button>
+      <Button colorScheme='blue'>Button</Button>
+    </Stack>
+  );
+};
+
+export default Login;
```
Comment (UmmarHamid) [inline components/Button/Login.tsx]: This component should not be in Buttons and should have a proper name

**Review by UmmarHamid — CHANGES_REQUESTED** [review]

**Review by UmmarHamid — APPROVED** [review]
LGTM! Thanks for the quick resolution

**Pull request merged**